addons/project/models/project_task.py

```diff
@@ -1271,6 +1272,15 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, mo
         )
         if self.stage_id:
             render_context['subtitles'].append(_('Stage: %s', self.stage_id.name))
+        if render_context['subtype'].name == 'Task Created':
+            render_context['message'].body = Markup("""
+                <ul style="margin-left: 12px; padding-bottom: 5px; margin-bottom: 20px;">
+                    <li style="font-size: 13px;">
+                        Project: %(project_name)s &#8594; Task Created
+                    </li>
+                </ul>
+                <hr style="background-color: rgb(204,204,204); border: medium none; clear: both; display: block; font-size: 0px; min-height: 1px; line-height: 0; margin: 10px 0px;">
+            """) % {'project_name': self.project_id.name}
```

Comment:
That is a little ugly :slightly_frowning_face: Isn't it possible to get the template?

Reply:
Yes, It is possible to get the template.